### PR TITLE
Luigi's contribution

### PR DESCRIPTION
**Pull request by ggx**
Hereafter a list of the main changes. IMHO with these changes the draft is in good shape to ask PEARG community if the document is going in the right direction. Hopefully it is and we can later add more text to the different sections. 

- Added several references
- Added subsection 3.3 on law issues
- Bootstrapped section 3.1
- Added text section 3.2
- Proposed content for 3.4
- Added text and revised section 4.1

### CODE REVIEW

**Comment by saradickinson** [comment]
@ggx Thanks for this PR! It seems Matthew did a PR just before you so we've merged his and there are now a few conflicts. Would you be able to take a look and fix those up?

File: draft-ip-address-privacy-considerations.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ---
 title: "IP Address Privacy Considerations"
 abbrev: "IP Address Privacy Considerations"
-docname: draft-ip-address-privacy-considerations
+docname: draft-ip-address-privacy-considerations-latest
```
Comment (saradickinson) [inline draft-ip-address-privacy-considerations.md:4]: I think we should leave the doc name as `draft-ip-address-privacy-considerations` unless you had a specific reason for changing it?

File: draft-ip-address-privacy-considerations.md [hunk 2]
```diff
@@ -71,50 +96,82 @@ This draft attempts to capture the following aspects of the tension between vali
...
+- GDPR (General Data Protection Regulation) - EUROPE: Europe considers IP addresses as personal identification information that should be treated like any other personal information e.g. social security number.
+- The United States has opted for a different approach to data protection. Instead of formulating one all-encompassing regulation such as the EU’s GDPR, the US chose to implement sector-specific privacy and data protection regulations that work together with state laws to safeguard American citizens’ data.
+- In 2020, China released the first draft of Personal Information Protection Law (PIPL). The PIPL is the equivalent of European GDPR and will have significant influence.
+- Japan Protection of Personal Information (APPI) Act (recent changes put the act close to the GDPR model).
 
-Commerical VPNs offer a layer of indirection between the user and the destination, however if the VPN endpoint's IP address is static then this simply substitutes one address for another. In addition, commerial VPNs replace tracking across sites with a single company that may track their users' activities.
+## Mitigations for IP address tracking
```
Comment (saradickinson) [inline draft-ip-address-privacy-considerations.md:131]: Most conflicts with Matthew's update are in this section

File: draft-ip-address-privacy-considerations.md (hunk 1 above)
Comment (chris-wood) [reply draft-ip-address-privacy-considerations.md:4]: I think the  "-latest" suffix is necessary for MT's tooling to produce the correct document version when compiling.

**Comment by ggx** [comment]
Hi,

I’ll have a look during the weekend

Ciao

L.

File: draft-ip-address-privacy-considerations.md (hunk 2 above)
Comment (ggx) [reply draft-ip-address-privacy-considerations.md:135]: Hi,

I went over the conflicts and solved them adjusting the text as needed.

Ciao

L.

**Comment by ggx** [comment]
Hi,

I solved all the conflicts adjusting the text as needed.
the repository is now up to date and clean.

Ciao

L.

**Pull request merged**